Chat/FileGetSocket.cpp

```diff
@@ -44,13 +44,9 @@ bool FileGetSocket::Initialize()
 
 unsigned FileGetSocket::GetFileStartup(void *this_ptr)
 {
-<<<<<<< HEAD
```

Comment:
be careful when you do commit. It is better to check all files two times to avoid such situations. If someone will see that in the future, it would cost a lot for you =(